modules/react/src/components/export-video/kepler/components/item-selector/chickleted-input.js

```diff
@@ -0,0 +1,143 @@
+// Forked from kepler.gl 11/2020
```

Comment:
Oh, we shouldn't need to fork in files anymore. I pushed a change to the upstream repo about this. See https://github.com/uber/hubble.gl/pull/45

Reply:
this is a file you deleted. I've been struggling to figure out why my merge from master didn't delete this file